src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
Don't leave empty lines between parent and child elements